Manifests/ManifestsApple/com.apple.domains.plist

```diff
@@ -246,6 +246,48 @@ SafariPasswordAutoFillDomains definitions are cumulative. Patterns defined by al
 			<key>pfm_type</key>
 			<string>array</string>
 		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Up to 10 domains can be added for which Cross-Site Tracking Prevention will be relaxed. Domains should be listed as theacmeinc.com which includes any subdomains (without needing to use *theacmeinc.com)</string>
+			<key>pfm_description_extended</key>
+			<string>As a result, organizations can leave Cross-Site Tracking Prevention turned on and benefit from tracking prevention for general browsing but also allow select domains to give third-party embedded resources the ability to use cookies. This is useful, for example, in education, where learning management systems embed content like videos or images stored by third parties, or learning tools (LTI tools) offered by third parties and presented in iFrames.</string>
+			<key>pfm_description_reference</key>
+			<string>Optional. An array of URL strings; maximum of 10. Supported in iOS 16.2 and later; Supported in macOS 13.1 and later. URLs matching the patterns listed here have relaxed enforcement of cross-site tracking prevention.</string>
+			<key>pfm_ios_min</key>
+			<string>16.2</string>
+			<key>pfm_macos_min</key>
+			<string>13.1</string>
+			<key>pfm_name</key>
+			<string>CrossSiteTrackingPreventionRelaxedDomains</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>iOS</string>
+				<string>macOS</string>
+			</array>
+			<key>pfm_repetition_max</key>
+			<integer>10</integer>
+			<key>pfm_subkeys</key>
+			<array>
+				<dict>
+					<key>pfm_description</key>
+					<string></string>
+					<key>pfm_name</key>
+					<string>CrossSiteTrackingDomain</string>
```

Comment:
If this is indeed an array of strings, `pfm_name` should be removed here.  `pfm_title` is the only thing you need besides the `pfm_type` and `pfm_value_placeholder`.  Remove the empty `pfm_description`

Reply:
Updating now

Reply:
Updated in 48cd1a0